### PR TITLE
enable SWI

### PR DESCRIPTION
**Pull request by dasvyat**
This is a small PR to enable SWI simulations in the master branch. There is a corresponding PR to ats-regression tests with  additional swi tests

### CODE REVIEW

File: src/pks/transport/transport_ats_pk.cc [hunk 1]
```diff
@@ -1256,6 +1257,12 @@ void Transport_ATS::AdvanceDonorUpwind(double dt_cycle)
             double tcc_flux = dt_ * u * values[i];
             (*conserve_qty_)[k][c2] += tcc_flux;
             mass_solutes_bc_[k] += tcc_flux;
+
+            if (tcc_tmp->HasComponent("boundary_face")) {
+              int bf = AmanziMesh::getFaceOnBoundaryBoundaryFace(*mesh_, f);
+              Epetra_MultiVector& tcc_tmp_bf = *tcc_tmp->ViewComponent("boundary_face",false);
```
[the remark this answers is not in the document]
Comment (dasvyat) [reply src/pks/transport/transport_ats_pk.cc]: Agreed. I've fixed it

[the remark this answers is not in the document]
Comment (dasvyat) [reply src/pks/transport/transport_ats_pk.cc]: I also removed wrong assumption on number of components in mpc_reactivetransport and mpc_coupled_reactivetransport.

File: src/pks/transport/transport_ats_pk.cc [hunk 2]
```diff
@@ -1248,6 +1250,10 @@ void Transport_ATS::AdvanceDonorUpwind(double dt_cycle)
       int c2 = (*downwind_cell_)[f];
       int c1 = (*upwind_cell_)[f];
 
+      if (tcc_tmp->HasComponent("boundary_face")) {
```
[the remark this answers is not in the document]
Comment (dasvyat) [reply src/pks/transport/transport_ats_pk.cc]: Fixed it

**Comment by hiter-joe** [comment]
Hi Daniil, 
I noticed that the latest status here is "Build and Test with Docker"
Does this mean we have the latest version of Docker?
Thanks!
Bing

**Pull request merged**